packages/ui/components/form/inputs/Input.tsx

```diff
@@ -23,7 +23,7 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
       {...props}
       ref={ref}
       className={classNames(
-        "hover:border-emphasis border-default bg-default placeholder:text-muted text-emphasis disabled:hover:border-default min-h-9 disabled:bg-subtle mb-2 block rounded-md border py-2 px-3 text-sm leading-4 focus:border-neutral-300 focus:outline-none focus:ring-2 focus:ring-neutral-800 focus:ring-offset-1 disabled:cursor-not-allowed",
```

Comment:
Anyone know why this had a min-height instead of a fixe height? Having them grow in size is risky when having addons, because changes are big they don't line up anymore.. All our inputs should have the same height, they shouldn't grow in height. If there's anything multiline, that's what textareas are for of course.